packages/backend/src/rafiki/request/asset.request.ts

```diff
@@ -0,0 +1,88 @@
+import {Asset, AssetEdge, AssetMutationResponse, AssetsConnection} from "../generated/graphql";
+import {gql} from "graphql-request";
+import {requestGQL} from "../graphql.client";
+
+export async function createAsset(
+    code: string,
+    scale: number,
+): Promise<Asset> {
+    const createAssetMutation = gql`
+    mutation CreateAsset($input: CreateAssetInput!) {
+      createAsset(input: $input) {
+        code
+        success
+        message
+        asset {
+          id
+          code
+          scale
+        }
+      }
+    }
+  `
+    const createAssetInput = {
+        input: {
+            code,
+            scale
+        }
+    }
+
+    return requestGQL<{ createAsset: AssetMutationResponse}>(createAssetMutation, createAssetInput, {})
```

Comment:
Like mentioned above, we can use `graphl-codegen` to also generate operation types directly ([typescript-operations](https://the-guild.dev/graphql/codegen/plugins/typescript/typescript-operations)).

Reply:
added a separate issue: https://github.com/interledger/testnet/issues/173